basyx.submodelservice/basyx.submodelservice.component/Readme.md

```diff
@@ -0,0 +1,99 @@
+# Eclipse BaSyx - Standalone Submodel Service Component
```

Comment:
general remark: this PR introduces two features: the SmService Component and the OperationDispatching. I would suggest rewriting the PR's title to mention the OperationDispatching feature or splitting the PR. I don't think 'Provide a generic submodel service component' summarizes the full scope of the changes.